packages/commerce-sdk-react/scripts/build-and-release-docs.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const sh = require('shelljs')
+const semver = require('semver')
+
+// this script is expected to run on release branch.
+const main = () => {
+    // get the version number from package json
+    const {stdout: currentVersion} = sh.exec(
+        "cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g'"
+    )
+    const isPrerelease = semver.prerelease(currentVersion.trim())
+    // if it is a pre-release version, do nothing
+    if (isPrerelease) {
+        return
+    }
+
+    let latestVersion
+
+    //TODO: Refactor this after we've release first version of commerce-sdk-react
+    // get the latest version from npm registry
+    const {stdout, stderr} = sh.exec('npm view commerce-sdk-react version', {silent: true})
```

Comment:
Although there is no harm in using the name of the package statically like this. It might be forward thinking to get the name from the above `packageJSON` variable and use it here in this command.